qa/rpc-tests/pegging.py

```diff
@@ -271,6 +288,47 @@ def write_bitcoin_conf(datadir, rpcport, rpcpass=None, p2p_port=None, connect_po
         if "confirmations" not in tx or tx["confirmations"] == 0:
             raise Exception("Peg-in confirmation has failed.")
 
+    print("Test pegout")
+    test_pegout(bitcoin.getnewaddress(), sidechain)
+
+    print("Test pegout Segwit (P2SH of witness script)")
+    parent_chain_addr = bitcoin.getnewaddress()
+    parent_chain_sw_addr = bitcoin.addwitnessaddress(parent_chain_addr)
```

Comment:
oops, this is deprecated in 0.16 and will be removed in 0.17.

In fact, getnewaddress will by default return a p2sh-p2wpkh address.

We could bump the mandatory version to 0.16 for this test, and explicitly set address types in the getnewaddress call.

Or we could set the default address at startup, older nodes will ignore it, and just make a 1-of-1 multisig addr that is p2sh and test that.